src/utils/Logger.ts

```diff
@@ -1,13 +1,20 @@
 const logPrefix = "LanguageWorkerConsoleLog";
 
 export function systemLog(message?: any, ...optionalParams: any[]) { 
-    console.log(logPrefix + message, ...optionalParams); 
+    console.log(logPrefix + removeNewLines(message), ...optionalParams); 
```

Comment:
We want to keep messages together whenever possible. a message sent to standard out / standard err with newlines will be treated as multiple, separate messages